dpbench/benchmarks/rodinia/gaussian/gaussian_numba_dpex_k.py

```diff
@@ -0,0 +1,105 @@
+# SPDX-FileCopyrightText: 2022 - 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import numba_dpex
+
+BLOCK_size_XY = 4
```

Comment:
Can this constant be moved to the config files?

Reply:
sycl implementation uses a fixed value for BLOCK_size_XY ,so we cannot make it configurable.

Reply:
Even more reason to move it to the config file. I see that there is a `#define` in the sycl implementation. Instead of maintaining them separately, create `BLOCK_size_XY` in the config file and pass in the same value to all implementation. Refer to `knn` workload, where we manage `k` in the config file.

Reply:
Made block_sizeXY configurable.